src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/service/MakeAnApplicationTypesProvider.java

```diff
@@ -266,7 +296,7 @@ && hasSubmittedHearingRequirements(asylumCase)) {
                 if (isAcceleratedDetainedAppeal(asylumCase) && hasRole(ROLE_LEGAL_REP)) {
                     values.add(new Value(TRANSFER_OUT_OF_ACCELERATED_DETAINED_APPEALS_PROCESS.name(),
                         TRANSFER_OUT_OF_ACCELERATED_DETAINED_APPEALS_PROCESS.toString()));
-                } else {
+                } else if (!isAcceleratedDetainedAppeal(asylumCase)) {
```

Comment:
Do we need to ask Elli that if the TRANSFER option is shown to any other user?

Reply:
After confirmation with Elli, the "Transfer" option is visible to all other user for non ADA cases after listing state (Prepare for hearing - decision state)